docs/notebooks/downstream_machine_learning_classification_evaluation.ipynb

```diff
@@ -0,0 +1,323 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Evaluate synthetic data on downstream classification models\n",
+    "\n",
+    "### How to use this notebook\n",
+    "Use this notebook to understand the quality and performance of your synthetic or augmented data on downstream machine learning classification tasks. \n",
+    "\n",
+    "### Installation\n",
+    "Install Gretel Client to use both Gretel's synthetic models as well as the Gretel Evaluate Classification model. You'll have to get your API key from the [Gretel console](https://www.console.gretel.ai) to configure your session. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pip install -U gretel-client"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from gretel_client import configure_session\n",
+    "\n",
+    "configure_session(endpoint=\"https://api.gretel.cloud\", api_key=\"prompt\", cache=\"yes\")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    " ## Try: Generate synthetic data, then evaluate the synthetic data on classifiers against real-world data\n",
+    " First, we'll generate synthetic data using a publicly available bank marketing dataset, which predicts whether a client will subscribe a term deposite (prediction: yes/no in column \"y\"). We'll use Gretel's LSTM model to train on the real-world and generate the synthetic data.\n",
```

Comment:
question: Do we want to give credit to the source of the data set (I'm guessing this is the bank datsaset from UCI ML repository).

Reply:
yup thanks for catching!